src/algorithm/geometry.hpp

```diff
@@ -91,6 +91,20 @@ namespace pinocchio
   bool computeCollision(const GeometryModel & geom_model,
                         GeometryData & geom_data,
                         const PairIndex & pairId);
+
+  ///
+  /// Calls computeCollision for every active pairs of GeometryData.
+  ///
+  /// \param[in] geom_model: geometry model (const)
+  /// \param[out] geom_data: corresponding geometry data (nonconst) where collisions are computed
+  /// \param[in] stopAtFirstCollision if true, stop the loop on pairs after the first collision.
+  ///
+  /// \warning if stopAtFirstcollision = true, then the collisions vector will
+  /// not be entirely fulfilled (of course).
+  ///
+  inline bool computeCollisions(const GeometryModel & geom_model,
+                                GeometryData & geom_data,
+                                const bool stopAtFirstCollision = true)
```

Comment:
@jcarpent I noticed that the two prototype of computeCollisions do not use the same default value for *stopAtFirstCollision*. Is this on purpose ?

Reply:
What is the second prototype?

Reply:
I see. No, this is not expected.

Reply:
I think both should be true by default. No?

Reply:
I would say so. For homogeneity, this should be changed. However, this may be extremely surprising for users and create a *hard to find* bug...

Reply:
Yes, indeed. I changed it, knowing we can revert it if needed. But the conservatism of default value true will pop up quickly I would say. Or we can use the default to be false, meaning that the behaviour will be correct for all the cases.

Reply:
Indeed, *false* ensures a correct behavior for all cases. It is safer. People caring for computation time should be more cautious.